README.md

```diff
@@ -35,6 +35,7 @@ It will not:
 
 ## Attributes
 
+* `['os-hardening']['container']['enable']` - true if this is a container. Default value is based on the information provided by ohai
```

Comment:
I think we need to make more clear what the impact of the setting is

Reply:
There is another PR #200. Should we maybe restructure default.rb in order to have a generic way to control which recipes should get included via default recipe? Something like this?

```ruby
# generic recipes
%w(packages limits).each do |rp|
  node.default['os-hardening']['recipes'][rp]['enable'] = true
end

# recipes which are not suitable for containers
unless node['virtualization']['system'] =~ /^(lxc|docker)$/ && node['virtualization']['role'] == 'guest' do
  node.default['os-hardening']['recipes']['sysctl']['enable'] = true
  node.default['os-hardening']['recipes']['autditd']['enable'] = true
end

# selinux recipe should be included only on RH based systems
node.default['os-hardening']['recipes']['selinux']['enable'] = 
  node['platform_family'] == 'rhel' || node['platform_family'] == 'fedora'
```

Reply:
Yes, this looks good and we keep the decision in default.rb